packages/eas-cli/src/user/SessionManager.ts

```diff
@@ -164,6 +167,19 @@ export default class SessionManager {
 
     Log.log('Log in to EAS');
 
+    // Prompt if sso has not been set (only login will have already set this value)
+    let useSso = sso;
+    if (useSso === undefined) {
+      useSso = await confirmAsync({
+        message: `Do you want to log in with SSO?`,
+      });
+    }
```

Comment:
I think we probably don't want to add this prompt here since all users will see it (adding friction to workflows). Instead, I think we may want to change the login prompt on L187 to say `Email or username (ctrl-c to exit and then run eas login for other login options):`

Reply:
Done